### DISCUSSION TITLE
R.S. Monitor Config

### DISCUSSION THREAD

**Discussion by fortizen** [cedricp/ddt4all#1293] [Q&A]
Hello, i saw some guy who put the R.S. Monitor on his Diesel Megane and he has the Turbo Pressure on it.

I have the R.S. Monitor installed on my Megane Trophy but no Turbo Pressure.

Is it possible to enable this or even another feature like the Traction Slip that the Megane 3 had;  with the RS Config string or another way?

**Comment by danikkm** [comment]
Perhaps @AndyIreland knows something?

**Reply by AndyIreland** [reply]
thanks for mentioning me , but I dont have a clue about the RS monitor or how you actually add it or any of the features to the r-link sorry
